src/ethereum_test_specs/blockchain.py

```diff
@@ -300,6 +300,11 @@ class BlockchainTest(BaseTest):
     genesis_environment: Environment = Field(default_factory=Environment)
     verify_sync: bool = False
     chain_id: int = 1
+    exclude_post_state: bool = False
```

Comment:
```suggestion
    exclude_full_post_state_in_output: bool = False
```
Just a nitpick on the naming of this field, just to make it explicit because otherwise I feel like it could be misinterpreted as excluding the post state in the input?

Reply:
Yes, and I also think this makes more clear the previous change. As in, the post-state assertions are still done, and this flag is only for avoiding putting it into the output.